virttest/utils_test/__init__.py

```diff
@@ -166,7 +166,7 @@ def __run_cmd_and_handle_error(msg, cmd, session, test_fail_msg):
 
 
 def update_boot_option(vm, args_removed="", args_added="",
-                       need_reboot=True, guest_arch_name='x86_64'):
+                       need_reboot=True, guest_arch_name='x86_64', serial=False):
```

Comment:
Add here `serial_login=False`

Reply:
@vivianQizhu Do you mean I need to adding "kernel_extra_params_serial_update = yes" in virtio_net_dpdk.cfg ?

Reply:
@quanwenli I guess so, if we have this in.

Reply:
@vivianQizhu @PaulYuuu current method defaults serial=True and why change it to False bydefault?
we can leave the default setting to true, as we anyway control it via param in other places needed?

Reply:
> @vivianQizhu @PaulYuuu current method defaults serial=True and why change it to False bydefault?
> we can leave the default setting to true, as we anyway control it via param in other places needed?

@sathnaga Thanks for your review. Actually, previously it's `False` by default, we just introduced serial login earlier this year here: https://github.com/avocado-framework/avocado-vt/pull/2564 for a specific network test case, for now only this case need `serial=True` and other test cases is okay with `False` by default.

Reply:
@sathnaga Changing the default value to False is due to some issues with serial login, please see https://github.com/avocado-framework/avocado-vt/pull/2828#issuecomment-729715574

Reply:
@sathnaga Any more comments? Thanks.

Reply:
@vivianQizhu LGTM,